charts/perses/templates/secrets.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.config.security.encryptionKeyFile .Values.config.security.encryptionKey }}
```

Comment:
We should provide the ability to users use an existing secret in the cluster.
- Disable the secret creation
- Provide the existing secret name and secret key to be mounted.

Reply:
Do you have any suggestions for what would it look like? I mentioned this in the PR's description but I'm not sure about the implementation. This gets a bit complex as there are many ways to make the key available to the perses process.

If `encryptionKey` is the only value provided then we set it in the config.yaml file.
If `encryptionKey` and `encryptionKeyFile` are set, we create the secret and mount it on the pod.
If `existingEncryptionKeySecret` (or any other name) is given we just set the volume and volume mount in the statefulset with the information provided by the user.
But what about precedence? For me, the following makes sense but let me now what you think. Existing secret > Encryption key mount as file > Encryption key put in the config.yaml file.

P.S.
I was thinking of the following format for the existing secret information to be provided.
```yaml
# -- Mount encryption key with an existing secret
existingEncryptionKeySecret:
  secretName: ""
  secretKey: ""
```

Reply:
I agree with the precedence you shared.

Note that you can't set `encryptionKey` and `encryptionKeyFile` at the same time in the Perses config otherwise the server will crashed because the config is inaccurate.

See https://github.com/perses/perses/blob/main/pkg/model/api/config/security.go#L55-L56